frontend/src/views/Game.vue

```diff
@@ -150,6 +166,17 @@ export default {
     const gameId = computed(() => gameStore.game.id?.toUpperCase());
     const isPlayerTurn = computed(() => gameStore.game.current_player === profileStore.getId);
 
+    const avatarUrl = computed(() => profileStore.avatarUrl);
+
+    const enemyId = computed(() => {
+      if (gameStore.game.first_player === profileStore.getId) {
+        return gameStore.game.second_player;
+      }
+      return gameStore.game.first_player;
+    });
```

Comment:
```suggestion
    const enemyId = computed(() => gameStore.game.first_player === profileStore.getId) ? gameStore.game.second_player : gameStore.game.first_player);
```

Tu peux utiliser une ternaire ici, c'est plus clean